### PR TITLE
ARROW-7876: [R] Installation fails in the documentation generation image

### PR DESCRIPTION
**Pull request by kszucs**
See build log https://github.com/apache/arrow/runs/452378996#step:5:11843

### CODE REVIEW

File: ci/scripts/cpp_build.sh [hunk 1]
```diff
@@ -129,6 +129,10 @@ time cmake --build . --target install
 
 popd
 
+if [ -x "$(command -v ldconfig)" ]; then
+  ldconfig
```
Comment (pitrou) [inline ci/scripts/cpp_build.sh]: Note that this won't work if non-root.

**Review by pitrou — APPROVED** [review]
+1

File: ci/scripts/cpp_build.sh (hunk 1 above)
Comment (kszucs) [reply ci/scripts/cpp_build.sh]: In docker we act as root, at least for now.

**Comment by kszucs** [comment]
Merging to resolve the failing CI on master.

**Pull request closed without merging**